memecity.game/Game/Systems/HudSystem.h

```diff
@@ -0,0 +1,29 @@
+﻿#ifndef HUDSYSTEM_H
```

Comment:
sinds de `run` hiervan empty is, denk ik nog steeds dat in plaats van een System, je deze hud updates direct als lambdas op de events kan registreren, zonder System.